.circleci/config.yml

```diff
@@ -40,6 +40,32 @@ jobs:
           name: Creator Dashboard tests
           command: scripts/tests.sh unlock-app
 
+  unlock-app-chromatic:
+    machine: true
+    environment:
+      DOCKER_REPOSITORY: unlockprotocol
+    steps:
+      - checkout
+      - run:
+          name: Pull Unlock Image
+          command: scripts/pull-image.sh unlock "$CIRCLE_SHA1"
+      - run:
+          name: Creator Dashboard stories
+          command: scripts/storybook.sh unlock-app
+
+  unlock-app-chromatic-master:
```

Comment:
I think rather than have 2 jobs you should push the logic of checking if we are on master down to `scripts/storybook.sh`

Reply:
You can pass the branch name using `$CIRCLE_BRANCH` and inside your script, if the branch is master, then you would append `--auto-accept-changes`.

This way we are as decouple as possible from the CI tool we use.

Reply:
thanks! this is the piece I needed.